### PR TITLE
DEVELOPER-1306 HOLD Change 'JBoss Champions' to 'JBoss Heroes'

### PR DESCRIPTION
**Pull request by Dantheman720**

### CODE REVIEW

**Comment by Dantheman720** [comment]
Currently testing links to make sure that changing from Champion to Hero didn't break code

File: _cdn/cdn.yml [hunk 1]
```diff
@@ -1409,13 +1409,13 @@ images_products_bpmsuite_bpmsuite.png:
 images_products_bpmsuite_rewards-workshop.png:
   build_no: 1
   md5sum: f8cc5660eaffa471bb3a59c0f01a7f3d
-images_champions_roadtoawesome_graphic.png:
```
Comment (paulrobinson) [inline _cdn/cdn.yml]: No need to make any changes to this file. The change in filename will cause cache invalidation.

File: _layouts/get-involved.html.slim [hunk 2]
```diff
@@ -81,11 +81,11 @@ ul.small-block-grid-1.large-block-grid-4
     a(href ="#{site.base_url}/video/vimeo/92868336/" )
       img(src=cdn(site.base_url + "/images/people/MarkusEisele.jpg") alt="Markus Eisele")
     div.get-involved-label
-      p.title Java Champion
+      p.title Java Hero
```
Comment (paulrobinson) [inline _layouts/get-involved.html.slim]: This needs to stay as 'Java Champion'. It's just the 'JBoss Champion' name that's changing.

File: _layouts/get-involved.html.slim [hunk 3]
```diff
@@ -81,11 +81,11 @@ ul.small-block-grid-1.large-block-grid-4
...
       p.name Markus Eisele
     blockquote It's more about a working living healthy community.
 
-    p We spoke with Markus Eisele about his role as a Java Champion and the JBoss community.
+    p We spoke with Markus Eisele about his role as a Java Hero and the JBoss community.
```
Comment (paulrobinson) [inline _layouts/get-involved.html.slim]: This needs to stay as 'Java Champion'. It's just the 'JBoss Champion' name that's changing.

File: events/javaland/speakers.html.slim [hunk 4]
```diff
@@ -36,7 +36,7 @@ table
...
+    p Arun Gupta is Director of Developer Advocacy at Red Hat and focuses on JBoss Middleware. A founding member of the Java EE team at Sun Microsystems, he led a cross-functional team at Oracle to drive the global launch of the Java EE 7 platform through strategy, planning, and execution of content, marketing campaigns, and program. Arun has extensive speaking experience in ~40 countries on myriad topics and is a JavaOne Rockstar. He also founded the Devoxx4Kids chapter in the USA and continues to promoting technology education amongst kids. An author of a best-selling book, an avid runner, a globe trotter, a Java Hero, JUG leader, he is easily accessible at @arungupta.
```
Comment (paulrobinson) [inline events/javaland/speakers.html.slim]: This needs to stay as 'Java Champion'. It's just the 'JBoss Champion' name that's changing.

**Comment by paulrobinson** [comment]
Can you do a search for 'Java Hero' and 'Java Heroes' and set them back to 'Java Champion(s)'. These are all casualties of the search-and-replace ;-)

Previously there where:

    Java Champions
    JBoss Champions

Now there are:

    Java Champions
    JBoss Heroes

**Comment by paulrobinson** [comment]
Summary from an email I sent around to coordinate:


Outstanding tasks:

1. Arun, can you review: http://www-stg.jboss.org/pr/862/build/1966/heroes/
2. James: Can you provide the new graphic for the banner?
3. Arun: Can you rename the twitter feed?
4. Paul: get a redirect in place for /champions ->/heroes

**Pull request merged**